.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,40 @@
+name: Bug Report
+description: Report a ouch bug
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for filling a bug report!
+
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: Version of ouch you encountered this error, or git commit hash if you are not using a versioned release
+    validations:
+      required: true
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+    validations:
+      required: true
+
+  - type: textarea
+    id: actual
+    attributes:
+      label: Actual Output
```

Comment:
I think it's better to call it `Current Behavior` and `Expected Behavior`, because some bugs are not directly related to "output".